src/main/java/org/mvc/controllers/MessagesController.java

```diff
@@ -0,0 +1,66 @@
+package org.mvc.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CookieValue;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+
+import org.mvc.handlers.MessagesHandler;
+import org.mvc.models.IncomingMessage;
+import org.mvc.models.SignIn;
+
+@RestController
+public class MessagesController {
+    @Autowired
+    private MessagesHandler messagesHandler;
```

Comment:
how would you inject an object into this field if it is private?. check contructor injection

Reply:
`constructor-based injection` added